test/units/TEST-87-TIMER.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+systemctl start realtime-test.timer
+
+sleep 35
+mindelta=10
+
+last=
+while read -r time; do
+    if [ -n "$last" ]; then
+        delta=$((time - last))
+        if [ "$delta" -lt $mindelta ]; then
+            echo "Timer fired too early: $delta < $mindelta" >/failed
+            break
+        fi
+    fi
+    last=$time
+done </tmp/realtime-test.log
+
+touch /testok
```

Comment:
hmm, can you plese make that a new subtest of TEST-74-AUX-UTILS instead of introducing a full-blown test for this, as they are kinda heavy on the CI.